draft-ietf-quic-recovery.md

```diff
@@ -955,15 +955,9 @@ Constants used in congestion control are based on a combination of RFCs,
 papers, and common practice.  Some may need to be changed or negotiated
 in order to better suit a variety of environments.
 
-kDefaultMss (default 1460 bytes):
-: The default max packet size used for calculating default and minimum
-  congestion windows.
-
-kInitialWindow (default 10 * kDefaultMss):
-: Default limit on the amount of outstanding data in bytes.
-
-kMinimumWindow (default 2 * kDefaultMss):
-: Default minimum congestion window.
+kInitialWindow (default 14600 bytes):
+: Default limit on the initial amount of outstanding data in bytes.
```

Comment:
I would note that RFC 6928 states the following as the TCP initial window:

   More precisely, the upper bound for the initial window will be

         min (10*MSS, max (2*MSS, 14600))                            (1)

I would recommend that one do calculate this value for the Initial value, as MSS may be based on a 9K MTU, thus allowing two full packets in those cases.

Reply:
Also if mss is smaller than 1460 then more than 10 packets can be sent. Is that intentional? I am slightly worried about packet bursts. Are there a pacing requirement?